src/native.js

```diff
@@ -305,8 +305,27 @@ export function fireImpressionTrackers(nativeResponse, {runMarkup = (mkup) => in
   }
 }
 
-export function fireClickTrackers(nativeResponse, {fetchURL = triggerPixel} = {}) {
-  (nativeResponse.link?.clicktrackers || []).forEach(url => fetchURL(url));
+export function fireClickTrackers(nativeResponse, {fetchURL = triggerPixel, assetId} = {}) {
```

Comment:
it's just style, but I'd prefer to either merge params together or keep dependencies separate from inputs (either `fireClickTrackers({nativeResponse, assetId, ...dependencies})` or `fireClickTrackers(nativeResponse, assetId, {...dependencies})` look better to me).

Reply:
I prefer the first one.